includes/wikia/nirvana/WikiaRequest.class.php

```diff
@@ -267,4 +267,18 @@ public function getScriptUrl() {
 		$scriptUrl = isset( $_SERVER['SCRIPT_URL'] ) ? $_SERVER['SCRIPT_URL'] : null;
 		return $scriptUrl;
 	}
+
+	/**
```

Comment:
is there an option to move this function to some helper class and call it from multiple places instead of copy/pasting?